packages/cli/src/constructs/construct.ts

```diff
@@ -4,9 +4,11 @@ import { Ref } from './ref'
 export abstract class Construct {
   type: string
   logicalId: string
+  __checkFilePath?: string
   constructor (type: string, logicalId: string) {
     this.logicalId = logicalId
     this.type = type
+    this.__checkFilePath = Session.checkFilePath
```

Comment:
Hmm, good point.

`main` is currently using this for `checkly test` and only assigning it for `Check` constructs. I think that it works in `main`, and we don't really need to change it. I think that most users will just define the checks directly in the  check file and there won't be many edge cases. I previously looked into getting the actual file where the construct is initialized and didn't see a nice solution.

Adding this to `CheckGroup` and `AlertChannel` like I do in this PR might be problematic, though. I think that it's much more likely these will be defined in a separate file and imported into multiple check files. 

I think the easiest solution is to just remove the source file from the `--preview` output. I can go ahead and change all of that back so that the output is:

```
$ npx checkly deploy --preview

Create:
    BrowserCheck: my-browser-check-logical-id
    BrowserCheck: another-browser-check

....
```

Reply:
I've updated the PR to remove all of the source file handling:

<img width="273" alt="Screenshot 2023-02-21 at 16 12 17" src="https://user-images.githubusercontent.com/10483186/220384482-592f3896-0e8e-4466-a70f-19f40c8d221f.png">